charts/kamus/templates/ingress.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.ingress.enabled -}}
 {{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
-apiVersion: networking.k8s.io/v1beta1
```

Comment:
This needs to be a more robust if/else structure.

```
{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
apiVersion: networking.k8s.io/v1
{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
apiVersion: networking.k8s.io/v1beta1
{{- else -}}
apiVersion: extensions/v1beta1
{{- end }}
```

You should open a PR against the upstream too.

Reply:
opened pr against upstream too. thanks for recommendation/logic